### PR TITLE
Add scripts for quick install

### PR DESCRIPTION
**Pull request by Tom-DaLin**
Through the two scripts provided here, users can quickly install swctl.

### CODE REVIEW

File: scripts/install.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Get the latest version number.
+VERSION=$(curl "https://raw.githubusercontent.com/apache/skywalking-website/master/data/releases.yml" | grep --after-context=7 "name: SkyWalking CLI" | grep "version" | grep -o "[0-9].[0-9].[0-9]")
+if [ "$VERSION" != "" ]; then
+    echo "Latest version: $VERSION"
+    # Download the package.
+    curl -LO "https://apache.claz.org/skywalking/cli/$VERSION/skywalking-cli-$VERSION-bin.tgz"
```
Comment (kezhenxu94) [inline scripts/install.sh:23]: We can simply use the mirrored address https://github.com/apache/skywalking-website/blob/93b417881ab4516346017ae10d8d2b3376ed3f22/data/releases.yml#L320

File: scripts/install.sh [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    if [ -f "skywalking-cli-$VERSION-bin.tgz" ]; then
+        # Verify the integrity.
+        curl -LO "https://downloads.apache.org/skywalking/cli/$VERSION/skywalking-cli-$VERSION-bin.tgz.sha512"
```
Comment (kezhenxu94) [inline scripts/install.sh:26]: We can simply use https://github.com/apache/skywalking-website/blob/93b417881ab4516346017ae10d8d2b3376ed3f22/data/releases.yml#L320-L324

File: README.md [hunk 3]
```diff
@@ -28,6 +28,23 @@ Then copy the `./bin/swctl-latest-(darwin|linux|windows)-amd64` to your `PATH` d
 usually `/usr/bin/` or `/usr/local/bin`, or you can copy it to any directory you like,
 and add that directory to `PATH`, we recommend you to rename the `swctl-latest-(darwin|linux|windows)-amd64` to `swctl`.
 
+In addition, you can also quickly install through the following methods:
+
+**Install swctl on Linux**
```
Comment (fgksgf) [inline README.md:33]: The script can also be used for MacOS, but need to add some judgment.

File: README.md [hunk 4]
```diff
@@ -28,6 +28,23 @@ Then copy the `./bin/swctl-latest-(darwin|linux|windows)-amd64` to your `PATH` d
...
+
+```shell
+sudo ./scripts/install.sh
```
Comment (fgksgf) [inline README.md:36]: You should assume that users don't clone the project, so the command should be like this: `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`

File: README.md [hunk 5]
```diff
@@ -28,6 +28,23 @@ Then copy the `./bin/swctl-latest-(darwin|linux|windows)-amd64` to your `PATH` d
...
+```
+
+**Install swctl on Windows**
+
+First you need to start cmd in administrator mode.
+
+```shell
+./scripts/install.bat
```
Comment (fgksgf) [inline README.md:44]: Ditto.

**Review by fgksgf — CHANGES_REQUESTED** [review]

File: scripts/install.sh (hunk 1 above)
Comment (Tom-DaLin) [reply scripts/install.sh]: But this is a web link, not a specific installation package download link. If I change like this, I will not be able to download the installation package.

File: README.md [hunk 6]
```diff
@@ -28,6 +28,24 @@ Then copy the `./bin/swctl-latest-(darwin|linux|windows)-amd64` to your `PATH` d
 usually `/usr/bin/` or `/usr/local/bin`, or you can copy it to any directory you like,
 and add that directory to `PATH`, we recommend you to rename the `swctl-latest-(darwin|linux|windows)-amd64` to `swctl`.
 
+In addition, you can also quickly install through the following methods:
+
+**Install swctl on Linux or macOS**
+
+```shell
+/bin/bash -c "$(curl -fsSL https://github.com/apache/skywalking-cli/tree/master/scripts/install.sh)"
```
Comment (kezhenxu94) [inline README.md:36]: This is not a valid URL https://github.com/apache/skywalking-cli/tree/master/scripts/install.sh

Comment (fgksgf) [reply README.md:36]: Because it is not merged yet, and should be https://raw.githubusercontent.com/apache/skywalking-cli/tree/master/scripts/install.sh

**Comment by wu-sheng** [comment]
This PR makes me feel, we will have `brew` install one day :)

**Comment by fgksgf** [comment]
> This PR makes me feel, we will have `brew` install one day :)

Yes, that's in our roadmap :) We just started with the basic installation method.

**Review by fgksgf — APPROVED** [review]
LGTM

**Review by kezhenxu94 — APPROVED** [review]
Beside the `brew` installation, installing via `go install apache/skywalking-cli` would be another interesting method to evaluate @Tom-DaLin @fgksgf

**Comment by fgksgf** [comment]
> Beside the `brew` installation, installing via `go install apache/skywalking-cli` would be another interesting method to evaluate @Tom-DaLin @fgksgf

Good idea.

**Pull request merged**